starknet_py/tests/e2e/docs/guide/test_using_existing_contracts.py

```diff
@@ -63,23 +68,23 @@ async def test_using_existing_contracts(run_devnet):
 
     # Using only positional arguments
     invocation = await contract.functions["transferFrom"].invoke(
-        sender, recipient, 10000, max_fee=0
+        sender, recipient, 10000, max_fee=MAX_FEE
```

Comment:
MAX_FEE is ambiguous from reader's standpoint. the content should be copy-pasteable as-is